charts/jaeger/templates/allinone-deploy.yaml

```diff
@@ -54,10 +54,19 @@ spec:
         {{- end }}
           securityContext:
             {{- toYaml .Values.allInOne.securityContext | nindent 12 }}
+          {{- if not .Values.v2.enabled }}
           image: {{ include "allInOne.image" . }}
+          {{- end }}
+          {{- if .Values.v2.enabled }}
+          image: {{ .Values.v2.image }}
+          {{- end }}
           imagePullPolicy: {{ .Values.allInOne.image.pullPolicy }}
           name: jaeger
           args:
+          {{- if .Values.v2.enabled }}
+            - "--config"
+            - "/etc/jaeger/config.yaml"
+          {{- end -}}
             {{- range $arg := .Values.allInOne.args }}
```

Comment:
I think this should only run if v2 not enabled

Reply:
when we enabled v2 we need to pass --config flag as arg and pass the conifg.yaml

Reply:
yes, but we cannot pass any other v1 flags, they will not be recognized